templates/index.html

```diff
@@ -282,7 +283,7 @@ <h3><span>Courses</span><small>Courses</small></h3>
                 </div>
 
                 <div class="col-lg-6 col-12">
-                    <h2 class="mb-4 mobile-mt-2">MSDS Align Program</h2>
+                    <h2 class="mb-4 mobile-mt-2">Data Science</h2>
```

Comment:
<img width="1336" alt="Screenshot 2023-09-25 at 8 17 42 PM" src="https://github.com/humans-of-align/humans-of-align-web/assets/62849837/42570b95-ba29-49ed-bcca-3837db161ad7">
Added section title and updated program titles